src/components/DeleteGameComponent/DeleteGameComponent.tsx

```diff
@@ -0,0 +1,71 @@
+import { gameSchema } from '@/utils/types';
+import {
+    ChakraProvider, 
+    AlertDialog,
+    AlertDialogBody,
+    AlertDialogFooter,
+    AlertDialogHeader,
+    AlertDialogContent,
+    AlertDialogOverlay,
+    AlertDialogCloseButton,
+    useDisclosure
+  } from '@chakra-ui/react'
+import { useRouter } from 'next/router';
+import React from 'react';
+
+interface Props {
+    gameName: string;
+}
+
+export default function DeleteGameDialog({ gameName }: Props) {
+    const gameID = useRouter().query.id;
+    const router = useRouter();
+    const { isOpen, onOpen, onClose } = useDisclosure();
+    const cancelRef = React.useRef();
+
+    async function deleteGame() {
+        fetch(`/api/games/${gameID}`,  {
+            method: 'DELETE'
+        });
+        router.push("/games");
+    }
+    
+    return (
+        <ChakraProvider>
+            <button onClick={onOpen} className="font-sans font-semibold bg-delete-red rounded-md text-white px-[17px] py-2 text-xl">
+                Delete Page
+            </button>
+            <AlertDialog
+                motionPreset='slideInBottom'
+                leastDestructiveRef={cancelRef}
+                onClose={onClose}
+                isOpen={isOpen}
+                isCentered
+            >
+                <AlertDialogOverlay />
+
+                <AlertDialogContent height="450" maxWidth="585">
+                    <AlertDialogCloseButton />
+                    <AlertDialogHeader>
+                        <div className="text-blue-primary text-[26px] font-bold mt-[114px] mx-[118px] text-center">
```

Comment:
there's a lot of white space above this text, could you make the modal smaller or move the text up?